scripts/pages/pgPeopleList.ts

```diff
@@ -0,0 +1,142 @@
+    }
+    async fetchPeople() {
+        try {
+            this.page++;
+            await peopleService.getAll(this.page);
+            this.servicePage++;
+            this.refreshListView();
+        } catch (error) {
+            console.log('fetchPeopleError', error)
+        }
+    }
+    initListView() {
+        this.lvPerson.rowHeight = LviPerson.getHeight();
+        this.lvPerson.onRowBind = (listViewItem: LviPerson, index: number) => {
+            listViewItem.name = store.getState().people.peopleList[index].name;
+            listViewItem.image = "images://darthvader.png";
+
+            listViewItem.flPersonWrapper.dispatch(
+                pushClassNames(index % 2 === 1 ? '.sf-listViewItem-active' : '.sf-listViewItem-inactive')
+            );
+
+            if (index + 1 < store.getState().people.peopleList.length) {
+                return; // Don't call service when scroll isn't at the bottom
+            }
+            if (this.page !== this.servicePage) {
+                return; // Prevent concurrent calls
+            }
+
+            this.fetchPeople();
+        };
+        this.lvPerson.onRowSelected = (_item: LviPerson, index: number) => {
+            this.router.push("/pages/pgPeopleDetail", { peopleIndex: index });
+        }
+        this.initListViewSwipe();
+        this.lvPerson.refreshEnabled = false;
+    }
+    initListViewSwipe() {
+        //@ts-ignore
+        this.lvPerson.swipeEnabled = true;
+
+        this.lvPerson.onRowCanSwipe = (index: number) => {
+            return [ListView.SwipeDirection.LEFTTORIGHT, ListView.SwipeDirection.RIGHTTOLEFT];
+        };
+        this.lvPerson.onRowSwipe = (e: any): ListView.SwipeItem[] => {
+            if (e.direction == ListView.SwipeDirection.LEFTTORIGHT) {
+                const someItem = new ListView.SwipeItem();
+                someItem.text = global.lang.someAction;
+                someItem.backgroundColor = Color.BLUE;
+                someItem.textColor = Color.WHITE;
+                someItem.onPress = ({ index }) => {
+                    this.lvPerson.refreshRowRange({ itemCount: 1, positionStart: index });
+                };
+                this.applyDimension(someItem);
+                return [someItem];
+            }
+            else if (e.direction == ListView.SwipeDirection.RIGHTTOLEFT) {
+                const detailItem = new ListView.SwipeItem();
+                detailItem.text = global.lang.detail;
+                detailItem.backgroundColor = Color.WHITE;
+                detailItem.textColor = Color.BLACK;
+                detailItem.onPress = ({ index }) => {
+                    this.lvPerson.refreshRowRange({ itemCount: 1, positionStart: index });
+                    this.router.push("/pages/pgPeopleDetail", { peopleIndex: index });
+                };
+                this.applyDimension(detailItem);
+                return [detailItem];
+            }
+        };
+    }
+    applyDimension(item: any): void {
```

Comment:
If you looked for a parent class for every UI component with child, you can use `import ViewGroup from 'sf-core/ui/viewgroup`. But it would be better to use ListViewItem